src/scripts.js

```diff
@@ -115,11 +121,46 @@ function postNewTrip(event){
         if(!response.ok) {
           throw new Error("Data failed to post");
         }
-        resolvePromises()
+        resolvePromises(currentUser.id)
         displayNewTripCost(newTrip)
         return response.json();
       })
 }
+
+function evaluateInformation(){
+    let userIdInput = username.value.split("r")[2]
+    let userId = parseInt(userIdInput)
+    let userTraveler = username.value.slice(0,8)
+    let passwordInput = pass.value
+    if(userTraveler === "traveler" && passwordInput === "travel" && userId <= 50 && userId){
+      resolvePromises(userId)
+      hideLogInPage()
+    } else if (userTraveler !== "traveler" && passwordInput !== "travel"){
+      invalidUsernameAndPassword()
+    } else if (userTraveler !== "traveler" || userId > 50 || !userId){
+      invalidUsername()
+    } else if (passwordInput !== "travel") {
+      invalidPassword()
+    }
+}
+
+function invalidUsernameAndPassword(){
+    wrongInputError.innerText = "Invalid username and password"
+}
+  
+function invalidUsername(){
+    wrongInputError.innerText = "Invalid username"
+}
+  
+function invalidPassword(){
+    wrongInputError.innerText = "Invalid password"
+}
```

Comment:
You can make this 1 function that accepts a message as a parameter and sets wrongInputError.innerText = message to DRY up the code a little, if you want.